lib/lookup.json

```
@@ -24,6 +24,37 @@
     "maintainers": "cjihrig",
     "prefix": "v"
   },
+  "@yarnpkg/cli": {
+    "envVar": { "TEST_IGNORE_TIMEOUT_FAILURES": true },
+    "install": ["install", "--mode", "skip-build"],
+    "maintainers": ["arcanis", "merceyz", "paul-soporan", "larixer"],
+    "prefix": "@yarnpkg/cli@",
+    "repo": "https://github.com/yarnpkg/berry",
+    "scripts": [
+      "build:cli",
```

```suggestion
      "build:cli --no-minify",
```
Should fix https://github.com/nodejs/citgm/pull/905#issuecomment-1280686628

---

Oh, this also works